tests/caasp/first_boot.pm

```diff
@@ -27,6 +27,12 @@ sub run {
     }
 
     microos_login;
+    if (check_var('DISTRI', 'kubic')) {
+        zypper_call("mr -da");
+        my $mirror = get_required_var('MIRROR_HTTP');
+        zypper_call("--no-gpg-check ar -f '$mirror' mirror_http");
+        zypper_call('ref');
+    }
```

Comment:
@rwx788 Can you check if this is a duplication of what we did in other modules already? The PR is fine as is but maybe we want to cleanup some duplication

Reply:
Oh yeah, we have same thing in kdump_utils, for url we have get_netboot_mirror in lib/utils